wdio.conf.js

```diff
@@ -0,0 +1,73 @@
+const localIP = require('ip');
+
+const staticServerPort = 4567;
+const AxeService = require('./lib/wdio/services').Axe;
+const TerraService = require('./lib/wdio/services').Terra;
+const visualRegression = require('./lib/wdio/visualcompare');
+
+// Force firefox into headless mode
+process.env.MOZ_HEADLESS = '1';
```

Comment:
Is this something we can setup in the config?

Reply:
this wouldn't matter anymore as we would run firefox within docker anyway